api/src/gameState/gameState.controller.ts

```diff
@@ -54,7 +61,7 @@ export class GameStateController {
       const game = await this.gameStateService.getGame({ id });
       return this.gameStateService.deserializerGame(game);
     } catch (error) {
-      throw new NotFoundException("That game does not exists");
+      throw new NotFoundException('That game does not exist');
```

Comment:
Same comment as for the `getGames`, also, Nest seems to handle automaticaly some exception for you, I tried to add a endpoint returning null, Nestjs automatically reponse a 404. I think, unless you need to throw a specific exception, you should let Nestjs pull all the wires for you ;) In other words : remove all the try catch if it's not handled by Nestjs already.